docs/articles/licensing/gplv3/gplv3-blacklist.rst

```diff
@@ -0,0 +1,86 @@
+GPLv3 licensed production software Vs automotive industry
+---------------------------------------------------------
+
+Modern cars have around 100 million lines of code running on them,12 with 70% 
+of that code being in the head unit. Complexity is a non-trivial issue in 
+automotive software design. In addition to being complex, cars can be dangerous.
+
+Locking the software to the hardware by signing the original software image
```

Comment:
Please add reference numbers to the mentioned references below, for example [1] [2] and use these numbers where ever you have taken the content from the referred articles. If you are using the exact words from these references then they should be clearly placed in the quotes.